domain/src/main/java/org/oppia/domain/topic/TopicController.kt

```diff
@@ -172,6 +175,27 @@ class TopicController @Inject constructor(
     )
   }
 
+  /** Returns the [ReviewCard] corresponding to the specified topic Id and subtopic ID, or a failed result if there is none. */
+  fun getReviewCard(topicId: String, subtopicId: String): LiveData<AsyncResult<ReviewCard>> {
+    return MutableLiveData(
+      try {
+        AsyncResult.success(retrieveReviewCard(topicId, subtopicId))
+      } catch (e: Exception) {
+        AsyncResult.failed<ReviewCard>(e)
+      }
+    )
+  }
+
+  // TODO(#21): Expose this as a data provider, or omit if it's not needed.
```

Comment:
Is this the right issue for this TODO? I'm not sure it relates to story progress. Perhaps: #45 would be clearer.

Reply:
Yeah, I mentioned it by mistake. Thanks for noticing. Updating it to #45.